The test failure looks real
```
======================================================================
FAIL: test_case_tile_suite (corehq.apps.app_manager.tests.test_suite.SuiteTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/test_suite.py", line 543, in test_case_tile_suite
    self._test_generic_suite("app_case_tiles", "suite-case-tiles")
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/test_suite.py", line 37, in _test_generic_suite
    self.assertXmlEqual(self.get_xml(suite_tag), app.create_suite())
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/util.py", line 55, in assertXmlEqual
    _check_shared(expected, actual, LXMLOutputChecker(), "xml")
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/util.py", line 100, in _check_shared
    raise AssertionError(message)
AssertionError: XML mismatch
--- want.xml
+++ got.xml
@@ -39,7 +39,7 @@
<stack>
<push>
<command value="'m0-f0'"/>
- <datum id="case_id_case_child" value="uuid()"/>
+ <datum id="case_id_new_rec_child_0" value="uuid()"/>
</push>
</stack>
</action>
----------------------------------------------------------------------
```

--- a/corehq/apps/app_manager/case_tile_templates/tdh.txt
+++ b/corehq/apps/app_manager/case_tile_templates/tdh.txt
@@ -14,7 +14,7 @@
       <stack>
       	<push>
       		<command value="'m0-f0'"/>
-      	    <datum id="case_id_case_child" value="uuid()"/>
+      	    <datum id="case_id_new_rec_child_0" value="uuid()"/>
       	</push>
       </stack>
     </action>

--- a/corehq/apps/app_manager/tests/data/suite/suite-case-tiles.xml
+++ b/corehq/apps/app_manager/tests/data/suite/suite-case-tiles.xml
@@ -40,7 +40,7 @@
       <stack>
       	<push>
       		<command value="'m0-f0'"/>
-      	    <datum id="case_id_case_child" value="uuid()"/>
+      	    <datum id="case_id_new_rec_child_0" value="uuid()"/>
       	</push>
       </stack>
     </action>

